data/classes/creature.cfg

```diff
@@ -758,10 +758,7 @@ properties: {
 	
 	move_creature: "def(class game_state game, int nmoves) ->commands
 	if(can_move_not_blocked(game),
-		if(nmoves >= 2 and game.creature_at_loc([loc.x, loc.y + direction_moving*2]) = null,
-		   do_move_internal(game, 2, direction),
-		   do_move_internal(game, 1, direction)
-		)
+	   do_move_internal(game, 1, direction)
```

Comment:
Exactly as you write:

> [...] I must admit that a long arrow is little confusing when a creature moves only one tile forward.

Me too would prefer that the action of the creature matches the drawing of the arrow.

If we included this change verbatim, players would learn of this change not as a result of a new short arrow appearing instead of (or additionally to) the current long arrow. But as the result of the gotcha of expecting a double movement and getting a single movement.

As [gotchas](https://en.wikipedia.org/wiki/Gotcha_(programming)) impact [usability](https://en.wikipedia.org/wiki/Usability), I'm against merging this change. 🙁

Reply:
Thanks for tips. I didn't know about branch creating rule - I'm not a programmer. I must learn more how to use github more properly.

I have a question. If I create a new branch, will it be possible to move this pull request there.
Because I have other ideas - I know how to make the arrow short all the time, but must figure out how to place information about remaining/used movement points.

Reply:
I didn't know you can change the _from_ branch of a pull request, but [this article promises it is possible](https://blog.github.com/2016-08-15-change-the-base-branch-of-a-pull-request/).

In case of any trouble with it, there is no problem in closing this pull request and referencing it from a new one for extended discussion.

Do as you please about this.

<hr />

However, let me reassert, if any development changes can be categorized as rule changes, I will not be able to merge them even if the code seems alright.